software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcessDriverLifecycleEffectorTasks.java

```diff
@@ -169,7 +173,7 @@ protected String startProcessesAtMachine(final Supplier<MachineLocation> machine
     }
 
     @Override
-    protected void postStartCustom() {
+    protected void postStartCustom(AtomicReference<ReleaseableLatch> startLatchRef) {
```

Comment:
We could force subclasses to error at compile time by leaving the removed methods in place and making them `final`. I'd prefer that to silently ignoring the method. Your suggestion of continuing to call the deprecated method seems like the least bad approach.